### PR TITLE
Add the Cinny client.

### PR DESCRIPTION
**Pull request by FieryMewtwo**

### CODE REVIEW

File: src/open/clients/Cinny.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    get homepage() { return "https://cinny.in"; }
+    get platforms() {
+        return [
+            Platform.DesktopWeb,
+        ];
+    }
+    get description() { return "Yet another Matrix client."; }
+    getMaturity(platform) {
+        switch (platform) {
+            case Platform.Android: return Maturity.Stable;
```
Comment (aaronraimist) [inline src/open/clients/Cinny.js:33]: You probably shouldn't put Android and iOS as Stable and then later say it doesn't support Android. Just leave those unsupported platforms out.

Comment (FieryMewtwo) [reply src/open/clients/Cinny.js:33]: oh whoops

**Comment by FieryMewtwo** [comment]
Sorry about my awful code -- I'm very new to JavaScript.

File: src/open/clients/Cinny.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        if (link.kind === LinkKind.User) {
+            switch (platform) {
+                case Platform.Android: return "Cinny doesn't support Android devices.";
+                case Platform.DesktopWeb: return "Open the web app at https://cinny.in and log in to your account. Click on 'Public Rooms' and paste the username.";
+                default: return "Open the app and click on 'Public Rooms' and paste the username.";
+            }
+        }
+        if (link.kind === LinkKind.Room) {
+            switch (platform) {
+                case Platform.Android: return "Cinny doesn't support Android devices.";
```
Comment (aaronraimist) [inline src/open/clients/Cinny.js:53]:
```suggestion
```

File: src/open/clients/Cinny.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    getInstallLinks(platform) {
+        switch (platform) {
+            default: return [new WebsiteLink("https://cinny.in")];
+        }
+    }
+
+    getLinkInstructions(platform, link) {
+        if (link.kind === LinkKind.User) {
+            switch (platform) {
+                case Platform.Android: return "Cinny doesn't support Android devices.";
```
Comment (aaronraimist) [inline src/open/clients/Cinny.js:46]:
```suggestion
```

File: src/open/clients/Cinny.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    getDeepLink(platform, link) {
+        switch (platform) {
+            case Platform.Android: return `in.fluffychat://`;
+            case Platform.iOS: return `in.fluffychat://`;
+            default: break;
+        }
+    }
+    canInterceptMatrixToLinks(platform) {
+        return platform === Platform.Android;
+    }
```
Comment (aaronraimist) [inline src/open/clients/Cinny.js:75]:
```suggestion
```

**Comment by FieryMewtwo** [comment]
I am so sorry; didn't expect this to take so long on my end.

**Comment by 3nt3** [comment]
can this be merged?

**Comment by thibaultamartin** [comment]
For some reason all the image files have been updated.
`/src/open/clients/index.js` should be updated too, so Cinny is added to the list of clients

**Comment by 3nt3** [comment]
I guess I'll just do my own PR unless OP, who seems to be inactive, fixes the files before me (@FieryMewtwo)

**Comment by thibaultamartin** [comment]
Duplicate of #286

**Pull request closed without merging**